third_party/move/move-compiler-v2/tests/checking/typing/bug_15274.exp

```diff
@@ -0,0 +1,41 @@
+
+Diagnostics:
+error: type `T` is missing required ability `store`
+   ┌─ tests/checking/typing/bug_15274.move:16:30
+   │
+16 │         unwrap_non_receiver(&Wrapper<T>[account])
+   │                              ^^^^^^^^^^
+   │
+   = required by instantiating type parameter `T:copy` of struct `Wrapper`
+   = required by instantiating type parameter `T:key` of function `borrow_global`
```

Comment:
Do we have any concern about users understanding why it's talking about `borrow_global` here?

Reply:
I think if users are using `Resource[addr]`, they should already know it is a syntactic sugar of borrow_global

Reply:
Ok.